src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -41,6 +41,60 @@ tags:
       </request>
       ```
       ... to this endpoint: `POST /request?cmd=create`
+
+
+      ## Workflow 2: Add a Review
+      **Add a User as Reviewer of a Submit Request**
+
+      First we make sure that a certain user can review our submit request.
+      We will add the user `test1` as a reviewer of the already created submit request.
+
+      For the sake of simplicity we suppose, that:
+        - There is an already created user, with login name `test1`.
+        - There is an already created submit request, with number `33`.
+
+      Adding the user `test1` as a reviewer of the submit request number `33` can be performed with this command:
+      `osc review add -m 'Add user test1 as reviewer' -U test1 33`
+
+      The same action can be done with this API endpoint: `POST /request/34?cmd=addreview&by_user=test1`
+
+      **Review a Submit Request as Accepted**
```

Comment:
Isn't this the same as "Accept a Submit Request Review"?

Reply:
"Mark a Review of a Submit Request as Accepted" would be enough I think

Reply:
Changed. Please review it again.